src/components/NftRewards/RewardsList/AddEditRewardModal.tsx

```diff
@@ -26,8 +26,8 @@ import {
 } from 'utils/antdRules'
 import { withHttps } from 'utils/externalLink'
 import { ipfsGatewayUrl } from 'utils/ipfs'
-import { CreateCollapse } from '../CreateCollapse'
-import { OptionalHeader } from '../OptionalHeader'
+import { CreateCollapse } from '../../Create/components/CreateCollapse'
+import { OptionalHeader } from '../../Create/components/OptionalHeader'
```

Comment:
**suggestion** if these aren't create-flow specific, might we move them out to the parent dir?

Reply:
Yes we should, was just keeping it mission critical for now - figured this would explode the diff.